### ISSUE TITLE
Mandates don't work in manually added subscriptions

### ISSUE THREAD

**Issue by rafaelzaleski** [woocommerce/woocommerce-gateway-stripe#2601]
**Describe the bug**
_Note: This issue only affects customer payments in INR_

When [subscriptions are added manually](https://woocommerce.com/document/subscriptions/add-or-modify-a-subscription/#section-1), they won't have a parent order and won't require customer interaction.

However, [mandates](https://stripe.com/docs/india-recurring-payments?integration=subscriptions#how-payments-work-with-an-e-mandate) require the customer to go through additional factor authentication (3DS) to register the mandate, so it will not be available for a manual subscription. 

To make mandates work with manual subscriptions, we need to explore alternative solutions, such as setting up the mandate with the first manual payment.

**To Reproduce**

Quoted from @reykjalin [comment](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2597#issuecomment-1510505507):

1. Create new subscription.
2. Add a customer to the subscription.
3. Add subscription valued at 15,000 INR or more to the subscription.
4. Choose `Manual renewal` for payment method.
5. Set subscription status to `Active`.
6. Click **Update** to create the subscription.
7. Follow the link in the subscription details to go to the customer's add payment method page and add a payment method there.
8. Go back to subscription details, select **Process renewal** from the available actions, and trigger that action by clicking the `>` button.
9. Look in my Stripe dashboard to see there is no mandate information passed on the payment intent request and no mandate is returned.

<img width="1030" alt="image" src="https://user-images.githubusercontent.com/13835680/232346089-3246710e-0e2b-4e4c-9273-93d2e68a5dd7.png">

Renewal payments work when following the eMandate testing flows from the 7.3.0 instructions, but **that's only because they're not sent through the mandate flow on Stripe's systems**.

**Expected behavior**
- We must support mandates on manually added subscriptions if it's technically possible.

**Screenshots**

**Environment (please complete the following information):**
 - WordPress Version: latest
 - WooCommerce Version: latest
 - Stripe Plugin Version: >= 7.3.0

**Additional context**
This is a follow-up issue from https://github.com/woocommerce/woocommerce-gateway-stripe/issues/2595 and the discussions on https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2597. @reykjalin already investigated some paths on https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2597, please read the comments on that PR before starting.
